### PR TITLE
Create pro

### PR DESCRIPTION
**Pull request by kajasharif05**
check this program

### CODE REVIEW

**Comment by kajasharif05** [comment]
can you see my program

**Comment by kajasharif05** [comment]
hii

**Comment by KameshvaranV** [comment]
good

**Comment by Abbas811** [comment]
hiii kaja super verified

**Comment by Abbas811** [comment]
1. hi no problem is solved

File: pro [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+ System.out.print(n1+" "+n2);//printing 0 and 1    
+    
+ for(i=2;i<count;++i)//loop starts from 2 because 0 and 1 are already printed    
+ {    
+  n3=n1+n2;    
+  System.out.print(" "+n3);    
+  n1=n2;    
+  n2=n3;    
+ }    
+
```
Comment (KameshvaranV) [inline pro:14]: good

**Review by KameshvaranV — COMMENTED** [review]
good

**Pull request merged**